mod/nodes/src/create_stream_action.go

```diff
@@ -0,0 +1,124 @@
+package nodes
+
+import (
+	"strings"
+
+	"github.com/cryptopunkscc/astrald/astral"
+	"github.com/cryptopunkscc/astrald/mod/exonet"
+	"github.com/cryptopunkscc/astrald/mod/nodes"
+	"github.com/cryptopunkscc/astrald/mod/scheduler"
+)
+
+var _ scheduler.Action = &CreateStreamAction{}
+
+type CreateStreamAction struct {
+	mod      *Module
+	Target   string
```

Comment:
field 'Target' should be *astral.Identity. generally speaking identity resolution should be done as early as possible.